frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditCodeList/EditCodeList.test.tsx

```diff
@@ -1,23 +1,25 @@
 import React from 'react';
 import { EditCodeList } from './EditCodeList';
-import { screen, waitFor } from '@testing-library/react';
+import { screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
 import { ComponentType } from 'app-shared/types/ComponentType';
-import { renderWithProviders, optionListIdsMock } from '../../../../../testing/mocks';
-import userEvent from '@testing-library/user-event';
+import { optionListIdsMock, renderWithProviders } from '../../../../../testing/mocks';
+import userEvent, { type UserEvent } from '@testing-library/user-event';
 import { textMock } from '@studio/testing/mocks/i18nMock';
 import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
 import type { FormComponent } from '../../../../../types/FormComponent';
-
-const mockComponent: FormComponent<ComponentType.Dropdown> = {
-  id: 'c24d0812-0c34-4582-8f31-ff4ce9795e96',
-  type: ComponentType.Dropdown,
-  textResourceBindings: {
-    title: 'ServiceName',
-  },
-  itemType: 'COMPONENT',
-  dataModelBindings: { simpleBinding: 'some-path' },
+import { componentMocks } from '@altinn/ux-editor/testing/componentMocks';
+import { addFeatureFlagToLocalStorage } from 'app-shared/utils/featureToggleUtils';
+import type { Option } from 'app-shared/types/Option';
+
+// Test data:
+const mockComponent: FormComponent<ComponentType.Dropdown> = componentMocks[ComponentType.Dropdown];
+mockComponent.optionsId = 'text';
```

Comment:
Maybe we can do like this:
```
const optionsIdMock = optionListIdsMock[0];
mockComponent.optionsId = optionsIdMock;
```
So we have a more real optionsId that we do not need to hardcode when we want to reference to it. F.ex as in the test below 🙈